src/FillArrays.jl

```diff
@@ -312,29 +317,30 @@ for (Typ, funcs, func) in ((:Zeros, :zeros, :zero), (:Ones, :ones, :one))
         @inline $Typ(::Type{T}, m...) where T = $Typ{T}(m...)
 
         @inline axes(Z::$Typ) = Z.axes
-        @inline size(Z::$Typ) = length.(Z.axes)
-        @inline getindex_value(Z::$Typ{T}) where T = $func(T)
+        # TODO: Should these be generic to `AbstractZeros`/`AbstractOnes`?
+        @inline size(Z::$AbsTyp) = length.(axes(Z))
```

Comment:
The reason for my comment was that I wasn't sure if there was a reason it was written `Z.axes` instead of `axes(Z)`, `axes(Z)` would assume less about the data structure of `AbstractZeros`/`AbstractOnes` subtypes.

Reply:
It should be `axes(Z)`